pages/categoryPage/category.html

```diff
@@ -2,10 +2,478 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Categories</title>
     <link rel="stylesheet" type="text/css" href="./category.css">
+    <script src="https://kit.fontawesome.com/5e5573fe57.js" crossorigin="anonymous"></script>
 </head>
 <body>
+<div class="header">
+    <div class="hamburgerMenu">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+    </div>
+
+    <div class="headerPhoneLogo">StoreNix</div>
+
+    <div class="headerList">
+
+        <div class="headerListElement">
+            <a class="headerTextLink">Home</a>
+        </div>
+        <div class="headerListElement">
+            <a class="headerTextLink">Categories</a>
+        </div>
+        <div class="headerListElement">
+            <a class="headerTextLink">Shop</a>
+        </div>
+
+        <div class="headerLogo">StoreNix</div>
+
+        <div class="headerListElement">
+            <a class="headerTextLink">Pages</a>
+        </div>
+        <div class="headerListElement">
+            <a class="headerTextLink">About</a>
+        </div>
+        <div class="headerListElement">
+            <a class="headerTextLink">Contact</a>
+        </div>
+
+    </div>
+
+    <div class="headerIcons">
+        <img class="headerIcon" src="../../assets/icons/bag-shopping.svg" alt="bag-shopping">
+        <img class="headerIcon" src="../../assets/icons/magnifying-glass.svg" alt="magnifying-glass">
+    </div>
+</div>
+
+
+<div class="informationBlock">
+    <div class="informationContainer">
+        <div class="mainText">
+            <p>New Arrivals</p>
+            <hr/>
+        </div>
+        <div class="pathText">
+            <p> Home <span class="pathSubText"> > </span>
+                New Arrivals <span class="pathSubText"> > </span>
+                <span class="pathSubText"> T-Shirts & Casuals </span> </p>
+        </div>
+    </div>
+</div>
+
+
+<div class="productsBlock">
+    <div class="overlay"></div>
+
+    <div class="filtersContainer">
+
+        <div class="filterBlock">
+            <p>CATEGORIES</p>
+            <ul>
+                <li><button onclick="openSubList('Arrivals')">
```

Comment:
add 'id' for button selector and after this add event listener in js file using this id